hypervisor/base/types.go

```diff
@@ -22,6 +22,8 @@ type Hypervisor interface {
 
 	Modify(vmName string, cpus int, mem int) error
 
+	GetVMProperty(vmName, property string) (*string, error)
```

Comment:
added this in hypervisor assuming we will be doing this with qemu as well to search for ip address